IngresseSDK/Services/EventService.swift

```diff
@@ -67,6 +67,25 @@ public class EventService: BaseService {
             onError(error)
         })
     }
+    
+    /// Get event image detail
+    ///
+    /// - Parameters:
+    ///   - eventId: id of the event
+    public func getEventImageDetail(ofEvent eventId: Int, onSuccess: @escaping (_ ads: Advertisement) -> Void, onError: @escaping (_ errorData: APIError) -> Void) {
```

Comment:
After this function there's another that do exactly the same thing that yours do. Do not duplicate functions with different names.